assets/src/dashboard/components/cardGridItem/test/cartTitle.js

```diff
@@ -41,7 +46,7 @@ describe('CardTitle', () => {
     const { getByTestId } = renderWithTheme(
       <CardTitle
         title="Sample Story"
-        displayDate="01/20/2020"
+        displayDate={moment('01/20/2020', 'MM/DD/YYYY')}
```

Comment:
lessons learned about how we need to format our dates, for sure! thank you for fixing this.